sycl/include/sycl/ext/oneapi/experimental/sycl_complex.hpp

```diff
@@ -979,6 +979,35 @@ tan(const complex<_Tp> &__x) {
   return complex<_Tp>(__z.imag(), -__z.real());
 }
 
+// Literal suffix for complex number literals [complex.literals]
```

Comment:
Corresponding [extension specification](https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/proposed/sycl_ext_oneapi_complex.asciidoc) should probably be updated as well to document that those literals should be supported and can be used